service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -15,23 +16,12 @@
 
 @Controller
 public class RunsApiController implements RunsApi {
+  private final SmartRunsPoller smartPoller;
   private final RunDao runDao;
 
-  public RunsApiController(RunDao runDao) {
+  public RunsApiController(RunDao runDao, SmartRunsPoller smartPoller) {
     this.runDao = runDao;
-  }
-
-  private RunState convertToRunState(String workflowStatus) {
-    return switch (workflowStatus) {
-      case "On Hold" -> RunState.PAUSED;
-      case "Submitted" -> RunState.QUEUED;
-      case "Running" -> RunState.RUNNING;
-      case "Aborting" -> RunState.CANCELING;
-      case "Aborted" -> RunState.CANCELED;
-      case "Succeeded" -> RunState.COMPLETE;
-      case "Failed" -> RunState.EXECUTOR_ERROR;
-      default -> RunState.UNKNOWN;
-    };
```

Comment:
These were Cromwell statuses, never returned by WES. WES statuses are an exact match for CBAS statuses... though it might be nice to separate out "API types" from "internal types" in case that statement slowly drifts away from truth